backend/app/spread/actions.py

```diff
@@ -30,25 +31,36 @@ def update_balances_with_spread(
     if token_to_divest_from == spread_token_symbol:
         return resized_balances
 
-    zeroes_series = make_zeroes(start, end)
+    def make_spread_token_unit_series():
```

Comment:
Can you explain this function? I can tell that it returns a `Series`, and that it divides the given price TS of the spread token by the initial price in the token's TS but I can't really tell why you're doing this. What is a unit series, and what does it achieve?

Reply:
It's the series of usd balances if you hold a constant balance of 1 token (like, 1 ETH or 1 USDC).

I'll try changing the name and I'll definitely add a comment.